### DISCUSSION TITLE
student verification failed.

### DISCUSSION THREAD

**Discussion by ghost** [community/community#58537] [GitHub Education]

**Comment by the-officialjosh** [comment]
In my experience, at my school, we were not normally provided with education mailboxes. However, I was able to resolve this issue by reaching out to the IT department of my institution. They were able to provide me with a temporary mailbox or alternative solution. I would suggest trying the same approach at your school. Contact the IT department and explain your situation, expressing your interest in applying for the GitHub Education package. Once you receive their response, we can explore other alternative solutions if necessary. I hope they are able to assist you in resolving this matter.
